main.go

```diff
@@ -159,13 +169,41 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Add readiness probe
+	//	Add readiness probe
 	if err := mgr.AddReadyzCheck("readyz", storagecluster.ReadinessChecker); err != nil {
-		setupLog.Error(err, "unable add a readiness check")
+		setupLog.Error(err, "unable to add a readiness check")
```

Comment:
This does not belong in this PR. Please remove.

Reply:
updated